### PR TITLE
missing descriptions for circuits

### PR DESCRIPTION
**Pull request by SonixApache**
special thanks to @DamianX, @ShiftyRail, and @Kurfursten.

### CODE REVIEW

**Comment by ShiftyRail** [comment]
Whoops

File: code/game/machinery/constructable_frame.dm [hunk 1]
```diff
@@ -1393,6 +1393,7 @@ obj/item/weapon/circuitboard/rdserver
 
 /obj/item/weapon/circuitboard/fishtank
 	name = "Circuit Board (Fishtank Filter)"
+	description = "A circuit board used to run a machine that holds fish"
```
Comment (jknpj) [inline code/game/machinery/constructable_frame.dm]: desc

File: code/game/machinery/constructable_frame.dm [hunk 2]
```diff
@@ -1401,6 +1402,7 @@ obj/item/weapon/circuitboard/rdserver
 
 /obj/item/weapon/circuitboard/fishwall
 	name = "Circuit Board (Large Fishtank Filter)"
+	description = "A circuit board used to run a machine that holds fish and acts as a wall"
```
Comment (jknpj) [inline code/game/machinery/constructable_frame.dm]: desc

File: code/game/machinery/constructable_frame.dm [hunk 3]
```diff
@@ -1409,6 +1411,7 @@ obj/item/weapon/circuitboard/rdserver
 
 /obj/item/weapon/circuitboard/conduction_plate
 	name = "Circuit Board (Conduction Plate)"
+	description = "A circuit board used to run a machine that can be placed under a fishtank to conduct electricity from electric eels"
```
Comment (jknpj) [inline code/game/machinery/constructable_frame.dm]: desc

File: code/game/machinery/constructable_frame.dm [hunk 4]
```diff
@@ -1417,6 +1420,7 @@ obj/item/weapon/circuitboard/rdserver
 
 /obj/item/weapon/circuitboard/holopad
 	name = "Circuit Board (Holopad)"
+	description = "A circuit board used to run a machine that allows AIs to project holograms in areas"
```
Comment (jknpj) [inline code/game/machinery/constructable_frame.dm]: desc

**Review by jknpj — CHANGES_REQUESTED** [review]

**Review by jknpj — APPROVED** [review]

**Pull request merged**